### ISSUE TITLE
Unexpected command error when using SSH

### ISSUE THREAD

**Issue by JirkaAichler** [zowe/zowe-cli#1031]
I am getting the following error that tells me to contact the support:

```
λ zowe zos-uss issue ssh "echo test"
Unexpected Command Error:
Please review the message and stack below.
Contact the creator of handler:
"C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\lib\zosuss\issue\ssh/Ssh.handler"
Message:
Invalid username
Stack:
Error: Invalid username
    at Client.connect (C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\node_modules\ssh2\lib\client.js:192:11)
    at C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\node_modules\@zowe\zos-uss-for-zowe-sdk\lib\Shell.js:102:18
    at new Promise (<anonymous>)
    at Function.executeSsh (C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\node_modules\@zowe\zos-uss-for-zowe-sdk\lib\Shell.js:33:25)
    at Handler.<anonymous> (C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\lib\zosuss\issue\ssh\Ssh.handler.js:38:57)
    at Generator.next (<anonymous>)
    at C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\lib\zosuss\issue\ssh\Ssh.handler.js:18:71
    at new Promise (<anonymous>)
    at __awaiter (C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\lib\zosuss\issue\ssh\Ssh.handler.js:14:12)
    at Handler.processCmd (C:\Users\ja628455\AppData\Roaming\npm\node_modules\@zowe\cli\lib\zosuss\issue\ssh\Ssh.handler.js:31:16)
```

**Steps to reproduce:**
My profiles do not contain a user name and password but contain an APIML JWT token.

Base:
```
  name:     sys1 (default)
  contents:
    host:               sys1.com
    port:               1443
    rejectUnauthorized: true
    tokenType:          apimlAuthenticationToken
    tokenValue:         1234
```

SSH:
```
  name:     sys1 (default)
  contents:
    host: sys1.com
    port: 22
```

```
λ zowe --version
6.32.0
```

**Comment by MarkAckert** [comment]
We should catch the error that's thrown in the SSH Handler and wrap it in an ImperativeError. 

Optionally, we can modify Imperative's error output for uncaught errors - I think the error content is clear enough as is, but we could remove the `Contact the creator of handler` line.

**Comment by adam-wolfe** [comment]
I am seeing similar behavior if I have no default SSH profile defined in a team config. (in 7.x.x and 8.0.0-next.202402271901)

**Comment by ATorrise** [comment]
Unsure why this linked issue wasn't closed with the merging of #2081 so am closing now
